styles.css

```diff
@@ -0,0 +1,214 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background: #222;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
```suggestion
  display: grid;
  Place-items: center;
```

Reply:
Yes this is a better way of stylinfg, I didn't use this because I thought that this doesn't work properly all the times.